src/Compiler/Microsoft.AspNetCore.Razor.Language/test/RazorDiagnosticTest.cs

```diff
@@ -90,22 +90,6 @@ public void DefaultRazorDiagnostic_ToString()
         Assert.Equal("test.cs(2,9): Error RZ0000: this is an error", result);
     }
 
```

Comment:
Why was this test removed?

Reply:
Good catch! That was unintended. Will add back